eZ/Publish/Core/MVC/Symfony/SiteAccess/Provider/StaticSiteAccessProvider.php

```diff
@@ -49,9 +56,22 @@ public function isDefined(string $name): bool
     public function getSiteAccess(string $name): SiteAccess
     {
         if ($this->isDefined($name)) {
-            return new SiteAccess($name, SiteAccess::DEFAULT_MATCHING_TYPE, null, self::class);
+            return $this->createSiteAccess($name);
         }
 
         throw new NotFoundException('Site Access', $name);
     }
+
+    private function createSiteAccess(string $name): SiteAccess
+    {
+        $siteAccess = new SiteAccess($name, SiteAccess::DEFAULT_MATCHING_TYPE, null, self::class);
+        $siteAccess->groups = array_map(
+            static function ($groupName) {
+                return new SiteAccessGroup($groupName);
+            },
+            $this->groupsBySiteAccess[$name]
```

Comment:
is it possible to set up a SA w/o a group? (don't remember). If so, what would happen here in this case?